AdobeStockClient/Model/SearchParametersProvider/Sorting.php

```diff
@@ -29,7 +29,7 @@ public function apply(SearchCriteriaInterface $searchCriteria, SearchParameters
     {
         $resultsOrders = Constants::getSearchParamsOrders();
         foreach ($searchCriteria->getSortOrders() as $sortOrder) {
-            if (false !== ($sortOrderField = array_search($sortOrder->getField(), $resultsOrders, null))) {
+            if (false !== ($sortOrderField = array_search($sortOrder->getField(), $resultsOrders, false))) {
```

Comment:
There is no need to pass optional parameters value, I think that's why they are optional :) Let's remove the last argument
```suggestion
            if (false !== ($sortOrderField = array_search($sortOrder->getField(), $resultsOrders))) {
```